cli/src/pcluster/validators/cluster_validators.py

```diff
@@ -760,7 +810,9 @@ def _validate(self, efs_id, avail_zones_mapping: dict, are_all_security_groups_c
             if head_node_target_id:
                 # Get list of security group IDs of the mount target
                 sg_ids = AWSApi.instance().efs.get_efs_mount_target_security_groups(head_node_target_id)
-                if not _check_in_out_access(sg_ids, port=2049, is_cidr_optional=are_all_security_groups_customized):
+                if not _is_access_allowed(
+                    sg_ids, subnets, port=2049, security_groups_by_nodes=security_groups_by_nodes
```

Comment:
Let's avoid magic n umbers if possible. Since 2049 is a notable port, let's define a self-explanatory constant for that

Reply:
Where would you put the constant? In the method, the file, or in constants.py?

Reply:
I suggest to put it in `constants.py` for the time being.
I think that `constants.py` must be split in smaller files with single responsibilities, but it is the best candidate for now.

Reply:
Done